common/azure/src/test/kotlin/com/cosmotech/api/azure/adx/AzureDataExplorerClientTests.kt

```diff
@@ -362,4 +363,21 @@ class AzureDataExplorerClientTests {
     val res = azureDataExplorerClient.deleteDataFromScenarioRunId("orgId", "wk", "my-scenariorunId")
     assertEquals("", res)
   }
+
+  @Test
+  fun `PROD-8987 - ingestScenarioValidationStatus`() {
+    every { csmPlatformProperties.azure?.dataWarehouseCluster?.options?.ingestionUri } returns
+        "https://ingest-cluster.location.kusto.windows.net"
+    every { csmPlatformProperties.azure!!.credentials.core.clientId } returns "my-clientId"
+    every { csmPlatformProperties.azure!!.credentials.core.clientSecret } returns "my-clientSecret"
+    every { csmPlatformProperties.azure!!.credentials.core.tenantId } returns "my-tentantId"
+
+    val azureDataExplorerClient =
+        spyk(AzureDataExplorerClient(csmPlatformProperties, eventPublisher))
+
+    assertDoesNotThrow {
```

Comment:
- Could you please add assertion on `ingestClient.ingestFromStream` calls?
- Could you please test that after X retry the loop is closed correctly?
The test right now will never throw an Exception and pass everytime...